resync/api/agents.py

```
@@ -3,72 +3,76 @@
-# Module-level dependency for agent manager to avoid B008 error
-agent_manager_dependency = Depends(get_agent_manager)
-
 agents_router = APIRouter()
 
 logger = logging.getLogger(__name__)
 
 
 @agents_router.get("/all")
 async def list_all_agents(
-    request: Request, agent_manager=agent_manager_dependency
+    request: Request, 
```

Ruff (0.15.1)

[warning] Unused function argument: `request`

(ARG001)